src/app/components/PromptInput.tsx

```diff
@@ -236,4 +236,4 @@ const PromptTextarea: React.FC<PromptTextareaProps> = ({
   );
 };
 
-export default PromptTextarea;
+export default PromptInput;
```

Comment:
initially it was textarea, but because later on this was changed to input forgot to rename